corehq/apps/ota/views.py

```diff
@@ -19,7 +19,7 @@
 from corehq import toggles
 from corehq.const import OPENROSA_VERSION_MAP
 from corehq.middleware import OPENROSA_VERSION_HEADER
-from corehq.apps.app_manager.util import get_app, LatestAppInfo
+from corehq.apps.app_manager.util import get_app_cached, LatestAppInfo
```

Comment:
This should be imported from dbaccessors. test failure related

Reply:
thanks, updated